src/azure-cli/azure/cli/command_modules/batch/tests/latest/test_batch_mgmt_commands.py

```diff
@@ -391,6 +378,23 @@ def test_batch_byos_account_cmd(self, resource_group):
                 self.check('name', '{byos_n}'),
                 self.check('location', '{byos_l}'),
                 self.check('resourceGroup', '{rg}')])
+        
+        # test for resource tags
+
+        self.cmd(
+            'batch account login -g {rg} -n {byos_n}'
+        )
```

Comment:
This command interferes with `batch account login` from `test_batch_general_arm_cmd` and causes failure when `test_batch_byos_account_cmd` and `test_batch_general_arm_cmd` are executed in parallel: https://github.com/Azure/azure-cli/issues/28500